Plutarch/Builtin/Bool.hs

```diff
@@ -18,21 +18,12 @@ module Plutarch.Builtin.Bool (
   pand',
   por',
   pcond,
+  ptrue,
+  pfalse,
```

Comment:
Why do we need these?

Reply:
we can't use `pconstant` in Builtin submodules. Having extra module solely for utility functions seemed too annoying